wagon.py

```diff
@@ -1011,6 +1011,94 @@ def repair(source, validate_archive=False):
     return archive_path
 
 
+def _combine_wheels(processed_source_a, processed_source_b, metadata_a,
+                    metadata_b):
+    """ The workdir of the new wagon is processed_source_a dir  """
+    wheels_path_a = os.path.join(processed_source_a, DEFAULT_WHEELS_PATH)
+    wheels_path_b = os.path.join(processed_source_b, DEFAULT_WHEELS_PATH)
+
+    for wheel in _get_downloaded_wheels(wheels_path_b):
+        if not os.path.isfile(os.path.join(wheels_path_a, wheel)):
+            logger.info("copying wheel {0}".format(wheel))
+            shutil.copy2(src=os.path.join(wheels_path_b, wheel),
+                         dst=wheels_path_a)
+            metadata_a["wheels"].append(wheel)
+    logger.info("finished combining wheels!")
+    return metadata_a
+
+
+def _compare_metadatas_field(metadata_a, metadata_b, field):
+    if not (metadata_a[field] == metadata_b[field]):
+        logger.info(
+            "{0} is different between accepted wagons,"
+            " cant combine them.".format(field))
+        return False
+    return True
+
+
+def _validate_combine(metadata_a, metadata_b):
+    fields_to_validate = ["build_server_os_properties",
+                          "created_by_wagon_version", "package_name",
+                          "package_version", "supported_platform"]
```

Comment:
consider moving this list to a constant instead of declaring inside the function.